### PR TITLE
getting job Artifacts added also added gitter and storyboard/sprintboard

### PR DESCRIPTION
**Pull request by akashshinde**

### CODE REVIEW

File: jenkins.go [hunk 1]
```diff
@@ -30,14 +31,18 @@ func NewJenkins(auth *Auth, baseUrl string) *Jenkins {
 }
 
 func (jenkins *Jenkins) buildUrl(path string, params url.Values) (requestUrl string) {
-	requestUrl = jenkins.baseUrl + path + "/api/json"
```
Comment (akashshinde) [inline jenkins.go]: need to find a better way to deal with url encoding

**Comment by akashshinde** [comment]
just need to replace gitter and waffle.io links with current repo

**Comment by yosida95** [comment]
Could you describe me why buildUrl is modified?  I didn't execute your branch, but this change seems breaking one for other methods.

**Comment by yosida95** [comment]
And the changes in README.rst (adding gitter and stroyboard/sprintboard) are not necessary for me.  Please revert these, or close the PR and create new PR with only changes to add getting job artifacts once again.

**Comment by akashshinde** [comment]
For getting artifacts url is `http://example.jenkins.com/job/job_name/api/json?tree=artifacts[*]`
so when I am trying to pass tree as key and value as `artifacts[*]` as value,encode method escaping [*]
so rest request was failing in that case,so I modified `buildURL` method to work with this scenario.

**Comment by yosida95** [comment]
For example, `(*Jenkins).GetQueue` needs to request to `http://example.jenkins.com/queue/api/json`, but in your branch, the URL modified to `http://example.jenkins.com/job/queue/api/json` and that endpoint not exists.

**Comment by akashshinde** [comment]
I can revert change back,but sometimes we need to provide parameters as `http://example.jenkins.com/job/job_name/api/json?tree=artifacts[*]`,but `url.Encode()`, replacing `[*]` so what do you think what is the best idea to work with such cases.

**Comment by akashshinde** [comment]
I am appending `/job/` to url path,so will remove that also

**Comment by bclermont** [comment]
@akashshinde can you please rebase your pull request?

I wish this could be merged. very useful

**Comment by bclermont** [comment]
> I wish this could be merged. very useful

I re-implemented it in a more simple way here:

https://github.com/yosida95/golang-jenkins/pull/19/files

@yosida95 is that better?

**Comment by yosida95** [comment]
Close this PR because artifacts feature is now supported in #19.

Thank you @akashshinde, @bclermont !

**Pull request closed without merging**